string task_name
---
string control_mode
bool success